src/plugins/intel_gpu/src/graph/graph_optimizer/fuse_primitives.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "program_helpers.h"
+#include "pass_manager.h"
+
+#include "eltwise_inst.h"
+#include "quantize_inst.h"
+
+#include <vector>
+#ifdef ENABLE_ONEDNN_FOR_GPU
+#include <impls/onednn/utils.hpp>
+#endif
+
+using namespace cldnn;
+
+static bool eltwise_supports_fusings(eltwise_node& node) {
+    auto out_layout = node.get_output_layout();
+    if (out_layout.data_type == data_types::f16 && out_layout.batch() > 1 && out_layout.format == format::fs_b_yx_fsv32) {
+        return false;
+    }
+
+    return true;
+}
+
```

Comment:
could you please add additional comment why this pass is additionally necessary?

Reply:
Added.